app/core/src/main/java/com/fsck/k9/preferences/AccountSettings.java

```diff
@@ -264,6 +264,15 @@
         s.put("trashFolderSelection", Settings.versions(
                 new V(54, new EnumSetting<>(SpecialFolderSelection.class, SpecialFolderSelection.AUTOMATIC))
         ));
+        s.put("resize_image_enabled", Settings.versions(
+                new V(56, new BooleanSetting(Account.DEFAULT_RESIZE_IMAGE_ENABLED))
```

Comment:
`Settings.VERSION` needs to be incremented when new settings are added. These entries should carry that new version number.